apps/expo-go/ios/Exponent/Versioned/Core/EXVersionManagerObjC.mm

```diff
@@ -257,33 +249,33 @@ - (void)selectDevMenuItemWithKey:(NSString *)key onBridge:(id)bridge
   }
 }
 
-- (void)showDevMenuForBridge:(id)bridge
+- (void)showDevMenuForHost:(id)host
 {
   RCTAssertMainQueue();
-  id devMenu = [self _moduleInstanceForBridge:bridge named:@"DevMenu"];
+  id devMenu = [self _moduleInstanceForHost:host named:@"DevMenu"];
   // respondsToSelector: check is required because it's possible this bridge
   // was instantiated with a `disabledDevMenu` instance and the gesture preference was recently updated.
   if ([devMenu respondsToSelector:@selector(show)]) {
     [((RCTDevMenu *)devMenu) show];
   }
 }
 
-- (void)disableRemoteDebuggingForBridge:(id)bridge
+- (void)disableRemoteDebuggingForHost:(id)host
 {
-  RCTDevSettings *devSettings = (RCTDevSettings *)[self _moduleInstanceForBridge:bridge named:@"DevSettings"];
+  RCTDevSettings *devSettings = [self devSettings:host];
   devSettings.isDebuggingRemotely = NO;
 }
 
-- (void)toggleRemoteDebuggingForBridge:(id)bridge
+- (void)toggleRemoteDebuggingForHost:(id)host
```

Comment:
maybe delete this since remote debugging is deprecated

Reply:
I will follow up with another PR for this